### PR TITLE
Platform 127

### PR DESCRIPTION
**Pull request by jamccomb92**
### Description of proposed changes
_What does this pull request accomplish? Why should it be integrated?_
Updated the sign up and sign in pages to use a formik component. Added sign up and sign in pages to initial application load. Added validation to the form inputs using Yup.
### Checklist
_Put an `x` in the boxes that apply. _
- [x ] My lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [ ] I have added or updated necessary documentation (if appropriate)

### Additional comments
Is there anything else you want us to know?
The validation schema will need to be refined and updated in the future.

### CODE REVIEW

File: screens/SignIn.js [hunk 1]
```diff
@@ -1,79 +1,73 @@
...
+const validationSchema = yup.object().shape({
+  username: yup
+    .string()
+    .label('Username')
+    .required(),
+  password: yup
+    .string()
+    .label('Password')
+    .required()
+    .min(7, 'Seems a bit short...')
```
Comment (hopetambala) [inline screens/SignIn.js:21]: yo! makes this 4 :D

Comment (hopetambala) [reply screens/SignIn.js:21]: we have users who make their passwords 4 and it's annoying af, but something we have to make sure we're being inclusive

File: screens/SignUp.js [hunk 2]
```diff
@@ -1,115 +1,123 @@
...
-            <TextInput
-              onChangeText={handleChange('phonenumber')}
-              onBlur={handleBlur('phonenumber')}
-              value={values.phonenumber}
-              style={styles.input}
+            <FormInput
+              label="Phone Number"
+              formikProps={formikProps}
+              formikKey="phonenumber"
+              placeholder="11118675309"
```
Comment (hopetambala) [inline screens/SignUp.js:98]: Yea just change this to `123-456-7890`

File: screens/SignIn.js (hunk 1 above)
Comment (jamccomb92) [reply screens/SignIn.js]: cool. adjusted

File: screens/SignUp.js (hunk 2 above)
Comment (jamccomb92) [reply screens/SignUp.js]: Done

**Pull request merged**